docs/loading/S3_Load.md

```diff
@@ -0,0 +1,272 @@
+
+## Using Broker Load
+
+An asynchronous Broker Load process handles making the connection to S3, pulling the data, and storing the data in StarRocks.
+
+### Advantages of Broker Load
+
+- Broker Load supports data transformation, UPSERT, and DELETE operations during loading.
+- Broker Load runs in the background and clients don't need to stay connected for the job to continue.
+- Broker Load is preferred for long running jobs, the default timeout is 4 hours.
+- In addition to Parquet and ORC file format, Broker Load supports CSV files.
+
+### Data flow
+
+![Workflow of Broker Load](../assets/broker_load_how-to-work_en.png)
+
+1. The user creates a load job
+2. The frontend (FE) creates a query plan and distributes the plan to the backend nodes (BE)
+3. The backend (BE) nodes pull the data from the source and load the data into StarRocks
+
+### Typical example
+
+Create a table, start a load process that pulls a Parquet file from S3, and verify the progress and success of the data loading.
+
+#### Create a table
+
+Create a database for your table:
+
+```SQL
+CREATE DATABASE IF NOT EXISTS project;
+USE project;
+```
+
+Create a table. This schema matches a sample dataset that you can load from a StarRocks S3 bucket.
+
+```SQL
+DROP TABLE IF EXISTS user_behavior;
+
+CREATE TABLE `user_behavior` (
+    `UserID` int(11),
+    `ItemID` int(11),
+    `CategoryID` int(11),
+    `BehaviorType` varchar(65533),
+    `Timestamp` datetime
+) ENGINE=OLAP 
+DUPLICATE KEY(`UserID`)
+DISTRIBUTED BY HASH(`UserID`) BUCKETS 1 
+PROPERTIES (
+    "replication_num" = "1"
+);
+```
+
+#### Gather connection details
+
+Loading data from S3 requires having the:
+
+- S3 endpoint
+- S3 region
+- Access key and secret
+
+#### Start a Broker Load
+
+This `LOAD` job has four main sections:
+
+- `LABEL`: A string used to query the state of a `LOAD` job.
+- `LOAD` declaration: The source URI, destination table, and the source data format.
+- `BROKER`: The connection details for the source.
+- `PROPERTIES`: Timeout value and any other properties to apply to this job.
+
+> Tip
+>
+> The dataset used in these examples is available in a StarRocks S3 bucket any valid `aws.s3.access_key` and `aws.s3.secret_key`. Substitute your credentials for `AAA` and `BBB` in the commands below.
+
+```SQL
+LOAD LABEL user_behavior
+(
+    DATA INFILE("s3a://starrocks-datasets/user_behavior_sample_data.parquet")
+    INTO TABLE user_behavior
+    FORMAT AS "parquet"
+ )
+ WITH BROKER
+ (
+    "aws.s3.enable_ssl" = "true",
+    "aws.s3.use_instance_profile" = "false",
+    "aws.s3.region" = "us-west-1",
+    "aws.s3.access_key" = "AAAAAAAAAAAAAAAAAAAA",
+    "aws.s3.secret_key" = "BBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB"
+ )
+PROPERTIES
+(
+    "timeout" = "72000"
+);
+```
+
+#### Check progress
```

Comment:
From v3.1 onwards, we provide a new table "loads" in the information_schema database, and we recommend the users to query job results from the loads table (for now this feature is supported only for Broker Load and Insert). SHOW LOAD is no longer an option, 'cause it does not return the path for storing info about filtered-out unqualified data rows. See https://docs.starrocks.io/en-us/3.1/loading/cloud_storage_load#view-a-load-job.

Reply:
I like this table, thanks for pointing that out!